src/Utilities/Compiler/DiagnosticCategoryAndIdRanges.txt

```diff
@@ -14,7 +14,7 @@ Globalization: CA2101, CA1300-CA1310
 Mobility: CA1600-CA1601
 Performance: HA, CA1800-CA1838
 Security: CA2100-CA2153, CA2300-CA2330, CA3000-CA3147, CA5300-CA5403
-Usage: CA1801, CA1806, CA1816, CA2200-CA2209, CA2211-CA2249
+Usage: CA1801, CA1806, CA1816, CA2200-CA2209, CA2211-CA2251
```

Comment:
Correct category is Performance as specified in https://github.com/dotnet/runtime/issues/33807